src/components/MarsApp.css

```diff
@@ -4,7 +4,7 @@
   background-repeat: no-repeat;
   background-position: center;
   background-size: cover;
-  height: 1060px;
+  height: auto;
 }
```

Comment:
We could make the background cover the whole screen no matter what size it is by adding some styling to the `html` element instead:

```
html { 
  background: url('../assets/images/background.jfif') no-repeat center center fixed;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
}
```